Andela-Ray/PropertyPro-lite
# #167270578 Update Repo - update README branch name

**What does this PR do?**
Corrects the branch name on the develop branch

**Description of Task to be completed?**
Update branch name to reflect correct test scores for develop branch, was erroneously using subsidiary feature/chore/bug branches

**How should this be manually tested?**
Visit the repo and view the README.md file on the develop branch, click on edit to see that the branch name for coveralls and travis is indeed develop, but don't edit it

**What are the relevant pivotal tracker stories?**
#167270578
